ci/jenkins/jobs/macros.yaml

```diff
@@ -293,6 +294,82 @@
           test_rc=$?
           set -e
 
+          echo "=== Test Prometheus client endpoint ==="
+          set +e
+          # Run and configure Prometheus
+          kubectl apply -f build/yamls/antrea-prometheus.yml
+
+          # Retrieve Prometheus credential for controller and agent access
+          certfile=$(mktemp /${WORKSPACE}/cacert.XXXXXX.ca)
```

Comment:
I think `${WORKSPACE}` will expand to an absolute path, so maybe no leading `/`.
Jenkins ensure parallel job runs in different workspace, so maybe we can use a fixed name for the cacert file, like `${WORKSPACE}/cacert-blah.ca`.